commands/release.go

```diff
@@ -195,6 +202,45 @@ func showRelease(cmd *Command, args *Args) {
 	os.Exit(0)
 }
 
+func downloadRelease(cmd *Command, args *Args) {
+	tagName := cmd.Arg(0)
+	if tagName == "" {
+		utils.Check(fmt.Errorf("Missing argument TAG"))
+	}
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	release, err := gh.FetchRelease(project, tagName)
+	utils.Check(err)
+
+	for _, asset := range release.Assets {
+		assetReader, apiError := gh.DownloadReleaseAsset(asset.ApiUrl)
+		utils.Check(apiError)
+		if assetReader == nil {
+			continue
+		}
+
+		ui.Printf("Downloading %s ...\n", asset.Name)
+
+		assetFile, err := os.OpenFile(asset.Name, os.O_WRONLY|os.O_CREATE|os.O_EXCL, 0600)
+		utils.Check(err)
+
+		_, err = io.Copy(assetFile, assetReader)
+		utils.Check(err)
+
+		assetFile.Close()
```

Comment:
Nitpick: maybe add `defer assetFile.Close()` right after opening the file so opening/closing logic is kept visually together?